ExploreASL/ExploreASL
# Bug #589 test data jsons

### Linked issue

Close #589

### How to test

Run the processing of the TestDataSet and verify that the GM CBF is ~50-60ml/min/100g
